src/styles/scss/tools/mixins.scss

```diff
@@ -0,0 +1,163 @@
+// Hide element visually but not from screen readers.
+// https://www.accessibility-developer-guide.com/examples/hiding-elements/visually/
+// ---
+// The class is intentionally called .hide-visually and not .visually-hidden.
+// Drupal which is a known consumer of the design system also has a
+// .visually-hidden class where the implementation is a bit different.
+// To avoid any conflicts we call our class something else.
+// //https://git.drupalcode.org/project/drupal/-/blob/9.2.x/core/modules/system/css/components/hidden.module.css#L18-33
+@mixin hide-visually {
```

Comment:
I agree, but SCSS is special.

```
// this wont show up in the compiled .css
/* this will show up in the compiled css */
```

it's done that way, so you can add stuff like font licensing, that should be visible.

Reply:
It does not have to be special.

1. As far as I can tell we do not have any licensing here that should be kept around.
2. We would switch to [compressed mode which will strip all comments and use `/*!` for comments that should not be stripped.](https://sass-lang.com/documentation/syntax/comments/#in-scss).

Reply:
Alright, I've set it to be compressed, and updated comments to use `/* */`